src/test/java/org/fundacionjala/coding/daniel/TestContarCaracteres.java

```diff
@@ -0,0 +1,52 @@
+package org.fundacionjala.coding.daniel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test that verifies how many times a character is repeated in a string.
+ */
+public class TestContarCaracteres {
+
+    private ContarCaracteres contar;
+
+    /**
+     * definition constructor of the CountCharacters.
+     */
+    @Before
+    public void setUp() {
+        contar = new ContarCaracteres();
+    }
+
+    /**
+     * Check how many times the character 'z' in the chain is repeated.
+     */
+    @Test
+    public void testRepiteCaracteres() {
+        final int expectedResult = 4;
+        int actualResult = contar.contarCaracter("fizzbuzz", 'z');
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define a constant instead of duplicating this literal "fizzbuzz" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)